activiti-cloud-audit-service/activiti-cloud-services-audit-jpa/src/test/resources/application.properties

```diff
@@ -1 +1,2 @@
-spring.application.name=audit
\ No newline at end of file
+spring.application.name=audit
+activiti.cloud.audit.uri-prefix=
```

Comment:
You should not need to have to define this in test properties if there is default properties source defined for each service